roles/vdm/defaults/main.yaml

```diff
@@ -99,6 +99,10 @@ V4_DEPLOYMENT_OPERATOR_SCOPE: cluster
 V4_DEPLOYMENT_OPERATOR_NAMESPACE: sasoperator
 V4_DEPLOYMENT_OPERATOR_CRB: sasoperator
 
+# SAS Orchestration Command
+# Experimental
+V4_SAS_ORCHESTRATION_USE_SITE_YAML: false
```

Comment:
For the use cases on this one. Do we have other parameters that are used for the Orchestration CLI or is this the only option we surface to customers? Why false. The goal here is to help folks move forward. By having set to false, the current customer support teams will not see any difference if they don't include this item. If we're supporting site.yaml it should always be there. The other option is to have the orchestration command line generated the sasdeployment.yaml file for use, but from all of the conversations around this one, I believe it's not needed if we're using site.yaml. Thoughts?

Reply:
Had a offline discussion with @thpang, updated this flag to "true"

So the updated behavior would be:

- Using the deployment operator as the deployment option (`V4_DEPLOYMENT_OPERATOR_ENABLED:true`) the site.yaml is still never generated. No change here.

- Using the sas-orchestration deploy command as the deployment option (`V4_DEPLOYMENT_OPERATOR_ENABLED:false`) the site.yaml will be generated and used (in the future) by default. The user **will** have to to turn this option off (`V4_SAS_ORCHESTRATION_USE_SITE_YAML:false`) if they instead want to use CRD with the sas-orchestration deploy command.

Not marking this as resolved, conversation planned with with the orchestration team for further clarifcation.

Reply:
Changing this back to false, we are **ONLY** generating the site.yaml at this point in time and not using it with the sas-orchestration deploy command. 

Since the file cannot actually be used by viya4-deployment and fed into the sas-orchestration deploy command until those changes are in place upstream, it makes sense to leave this false for now.

When we are able to use the site.yaml file with the sas-orchestration deploy we can considering toggling this true by default.

@thpang let me know if you agree.